mittab/apps/tab/pairing_views.py

```diff
@@ -211,6 +211,8 @@ def view_round(request, round_number):
     errors, excluded_teams = [], []
     round_pairing = list(Round.objects.filter(round_number=round_number))
 
+    tot_rounds = TabSettings.get("tot_rounds", 5)
```

Comment:
Whats the purpose of this change?

Reply:
This is so that on the pairings view the button can change to do the break when we are at the final round.  The template didn't know what the total number of rounds was before.